src/SFA.DAS.AdminService.Web/Infrastructure/RoatpApplicationApiClient.cs

```diff
@@ -143,6 +146,32 @@ public async Task<Guid> SnapshotApplication(Guid applicationId, Guid snapshotApp
             return await Post<SnapshotApplicationRequest, Guid>($"/Application/Snapshot", new SnapshotApplicationRequest { ApplicationId = applicationId, SnapshotApplicationId = snapshotApplicationId, Sequences = sequences });
         }
 
+        public async Task<CompaniesHouseSummary> GetCompanyDetails(string companiesHouseNumber)
```

Comment:
Might be cleaner to have calls to external APIs (even if via an internal proxy) in a separate API client class

Reply:
Good idea. Will do in next story